README.md

```diff
@@ -27,6 +27,13 @@ The buildpack will do the following for PHP applications:
 * Contributes a PHP agent to a layer and configures `$PHP_INI_SCAN_DIR` to use it
 * Transforms the contents of the binding secret to environment variables with the pattern `NEW_RELIC_<KEY>=<VALUE>`
 
+The buildpack will do the following for Python applications:
+
+* Copies configuration file in /resources/newrelic.ini to the root folder of your application
+* Installs New Relic Python agent using pip3
```

Comment:
As it stands now the buildpack isn't going install New Relic anymore which is fine. However, it's up to the user to make sure their requirements.txt file is set up to include the New Relic agent. You might want this buildpack, which will run after Python & PIP have run, do something to confirm that the agent has been installed. You could have it fail the build with a helpful error if the agent isn't present.

It might also be helpful to have a note in this README indicating that it's a requirement for the user to add this.

Reply:
Done